config/initializers/has_cloudinary_image.rb

```diff
@@ -65,10 +73,11 @@ def has_image?
 
       # return 100px square default image
       def image_url(options={})
-        image = cloudinary_id
+        image = cloudinary_id || "upload/#{randomized_image(rand(18))}"
         options = {
           size: 100,
-          default: "upload/noaoqqwxegvmulwus0un.png" # big gray B
+          # default: "upload/noaoqqwxegvmulwus0un.png" # big gray B
```

Comment:
We want to keep this for broken links. Sometimes users will link an image that no longer exist. Good to keep default so we can detect the problem